firmware/src/pdu.c

```diff
@@ -68,13 +68,35 @@ main(void)
 	/* ============================================================== */
 
 
+	int counter_inrush[6] = {0,0,0,0,0,0};
```

Comment:
These should be unsigned, and even though the size of `int` on avr-gcc is 16 bits, I personally prefer the explicitness of `uint16_t`.